db/migrate/migration_utils/utils.rb

```diff
@@ -82,6 +82,10 @@ def mysql?
       ActiveRecord::Base.connection.instance_values['config'][:adapter] == 'mysql2'
     end
 
+    def sqlite3?
+      ActiveRecord::Base.connection.instance_values["config"][:adapter] == "sqlite3"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.